src/renderer/codeMirror/index.css

```diff
@@ -4,4 +4,153 @@
 }
 .CodeMirror-line::selection, .CodeMirror-line > span::selection, .CodeMirror-line > span > span::selection {
   background: rgb(178, 215, 254);
-}
\ No newline at end of file
+}
+.CodeMirror-search-label{
+  
+}
+.CodeMirror-dialog {
```

Comment:
Could you please check the styling with dark themes, it's a bit odd. Also, OneDark has dark font color on dark background.

Reply:
Yes the control was essentially un-themed.  Would you prefer legibility is working for both, or to actually flip the theme based on marktext?

Reply:
Flipping the theme would be great. I think we use two different themes based whether MarkTexts theme is light or darkish. Maybe we can use CSS selectors to differentiate colors.

Reply:
Yeah I assumed CSS light/dark overall would be the way to go. Will update.